src/config/logger.test.ts

```diff
@@ -1,64 +1,43 @@
-import { getLogger, getUnsafeLogger } from "config/logger";
```

Comment:
This looks like a lot of changes in this file, but really I've just removed a bunch of tests for serialization and moved them to logger-utils.tests